### PR TITLE
Add may papercuts

### PR DESCRIPTION
**Pull request by assuntad23**
Creating directory for May papercuts event, although date may change, and the page needs more details, depending on which working group is hosting it.

### CODE REVIEW

**Comment by NickSto** [comment]
I need to document this better, but FYI you can add `draft: true` to the metadata to keep a post from being listed until it's ready.

**Comment by assuntad23** [comment]
@NickSto  I look forward to seeing that documented further. I'm not sure how to access that...

**Comment by NickSto** [comment]
Oh I mean literally just add the line
```
draft: true
```
to the top of the `index.md` file, like right under the title or date line.

**Comment by assuntad23** [comment]
@NickSto oooh! the actual metadata of the file you're editing! That's awesome. I thought you meant some GitHub metadata & I was like...  🤔

File: content/events/2022-05-papercuts/index.md [hunk 1]
```diff
@@ -0,0 +1,83 @@
...
+* Identify easy but informative issues to fix.
+* Work together and enjoy the community.
+* Dedicate time together for bug-fixing.
+* Make reducing paper cuts fun.
+
+This will be a 24h event, spanning all time zones with our worldwide community. We'll start at 09:00 Melbourne time, and end at 16:00 Portland time.
+
+## Interested?
+
+On Papercuts Day, please __join us on [Gitter](https://gitter.im/galaxyproject/Lobby)__ (all day). If you don't have a GitHub or Gitter (or Elements/Matrix) account yet, then set them up (in that order).
```
Comment (beatrizserrano) [inline content/events/2022-05-papercuts/index.md:38]:
```suggestion
On Papercuts Day, please __join us on [Gitter](https://gitter.im/galaxyproject/Lobby)__ (all day). If you don't have a GitHub or Gitter (or Element/Matrix) account yet, then set them up (in that order).
```

File: content/events/2022-05-papercuts/index.md [hunk 2]
```diff
@@ -0,0 +1,83 @@
...
+# During the event
+
+
+We will be on [Gitter](https://gitter.im/galaxyproject/Lobby) for chat all day long.
+
+There are also two GTN CoFest calls planned for the day.
+
+
+* **[GTN CoFest](https://github.com/galaxyproject/training-material/issues/1712) Call 1: Middle East, Europe, Africa**
+    * 10:00 Central European time.  [See your time](https://www.timeanddate.com/worldclock/fixedtime.html?msg=EMEA+GTN+CoFest+Call&iso=20220519T10&p1=980&am=30).
```
Comment (beatrizserrano) [inline content/events/2022-05-papercuts/index.md:63]:
```suggestion
    * 10:00 Central European Summer time.  [See your time](https://www.timeanddate.com/worldclock/fixedtime.html?msg=EMEA+GTN+CoFest+Call&iso=20220519T10&p1=980&am=30).
```

File: content/events/2022-05-papercuts/index.md [hunk 3]
```diff
@@ -0,0 +1,83 @@
...
+
+
+## After the event
+
+*Get those pull requests in! And address anything from the review! And become more involved in the Galaxy code, training, and documentation base!*
+
+
+## Want to host?
+
+If you're a member of a working group and would like to nominate yourselves to host a papercut event, please contact Assunta DeSanto.
```
Comment (beatrizserrano) [inline content/events/2022-05-papercuts/index.md:83]: Maybe a preferred way to contact you? Or not needed?

**Review by beatrizserrano — COMMENTED** [review]
Looks good to me! And the issue of not showing up on the list of upcoming events seems to be fixed when adding subsites :)

File: content/events/2022-05-papercuts/index.md (hunk 3 above)
Comment (assuntad23) [reply content/events/2022-05-papercuts/index.md:83]: If it's just internal, they can find me...

**Review by beatrizserrano — APPROVED** [review]

**Pull request merged**